homeassistant/components/huisbaasje/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unauthenticated_exception": "[%key:common::config_flow::error::invalid_auth%]",
+      "connection_exception": "[%key:common::config_flow::error::cannot_connect%]",
```

Comment:
Please use the standard key names eg `"cannot_connect"` in the object besides the references.